flutter/pubspec.yaml

```diff
@@ -1,6 +1,6 @@
 name: fluentui_system_icons
 description: Fluent UI System Icons are a collection of familiar, friendly and modern icons from Microsoft.
-version: 1.1.82
+version: 1.1.83
```

Comment:
This should remain as 82? Because the publish script will auto increment and commit it.